src/eve/traits.py

```diff
@@ -68,3 +71,25 @@ def _collect_symbols_validator(  # type: ignore  # validators are classmethods
     def collect_symbols(self) -> None:
         self.symtable_ = dict()
         self.symtable_ = self._collect_symbols(self)
+
+    @staticmethod
+    @contextmanager
+    def add_symtable(
```

Comment:
`add_symtable` name is fine but since this is now a context manager, maybe `symtable_merger` or something like that would be clearer.

Reply:
👍